python/semantic_kernel/connectors/ai/mistral_ai/prompt_execution_settings/mistral_ai_prompt_execution_settings.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import logging
+from typing import Any, Literal
+
+from pydantic import Field
+
+from semantic_kernel.connectors.ai.prompt_execution_settings import PromptExecutionSettings
+
+logger = logging.getLogger(__name__)
+
+
+class MistralAIPromptExecutionSettings(PromptExecutionSettings):
+    """Common request settings for MistralAI services."""
+
+    ai_model_id: str | None = Field(None, serialization_alias="model")
+
+
+class MistralAIChatPromptExecutionSettings(MistralAIPromptExecutionSettings):
+    """Specific settings for the Chat Completion endpoint."""
+
+    response_format: dict[Literal["type"], Literal["text", "json_object"]] | None = None
+    tools: list[dict[str, Any]] | None = Field(None, max_length=64)
```

Comment:
We may want to remove the ability to set tools, or throw some validation error if they are set, since we don't support mistral function calling yet.